### PR TITLE
[14.0] Use tags to detect procedures

### LINKED ISSUE #518: Improvement on detection of Procedures
Right now, the only way to detect procedures is to use a single category created as data. It is a problem when the environment is complex, as a simple category for all your procedures might be hard to manage.

I have two ideas for that:

1- Adding them as `type`of document.page (some changes might be necessary, but not too much)
2- Create a subtype

Obviously, some migrations scripts are necessary, but I might be able to do that :smile: 

This could be used for procedures, work instructions, quality manuals...

@max3903 WDYT?

### PR DESCRIPTION
**Pull request by etobella**
Includes migration scripts

### CODE REVIEW

**Comment by etobella** [comment]
@max3903

**Comment by max3903** [comment]
@etobella I meant to use tags but also to keep the category since it also provides the content template.

File: document_page_environmental_aspect/data/document_page.xml [hunk 1]
```diff
@@ -1,26 +1,8 @@
...
-
-<h1>Responsibilities</h1>
-
-<h1>Objectives</h1>
-
-<h1>Programs</h1>
-        ]]>
-      </field>
+    <record id="environment_aspect_tag" model="document.page.tag" forcecreate="1">
+        <field name="name">Environment Aspect</field>
```
Comment (max3903) [inline document_page_environmental_aspect/data/document_page.xml]: @etobella I would keep the original file as it was and add a new one `document_page_tag.xml` with this content.

File: document_page_environmental_aspect/__manifest__.py [hunk 2]
```diff
@@ -25,7 +25,11 @@
     "website": "https://github.com/OCA/management-system",
     "license": "AGPL-3",
     "category": "Generic Modules/Others",
-    "depends": ["document_page", "mgmtsystem"],
-    "data": ["data/document_page.xml", "views/document_page.xml"],
+    "depends": ["document_page_tag", "mgmtsystem"],
+    "data": [
+        "data/document_page.xml",
+        "data/document_tag.xml",
```
Comment (max3903) [inline document_page_environmental_aspect/__manifest__.py]:
```suggestion
        "data/document_page_tag.xml",
```

File: document_page_procedure/__manifest__.py [hunk 3]
```diff
@@ -3,13 +3,17 @@
...
     "author": "Savoir-faire Linux, Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/management-system",
     "license": "AGPL-3",
     "category": "Management System",
-    "depends": ["document_page", "mgmtsystem"],
-    "data": ["data/document_page_procedure.xml", "views/document_page_procedure.xml"],
+    "depends": ["document_page_tag", "mgmtsystem"],
+    "data": [
+        "data/document_page_procedure.xml",
+        "data/document_tag.xml",
```
Comment (max3903) [inline document_page_procedure/__manifest__.py]:
```suggestion
        "data/document_page_tag.xml",
```

**Review by max3903 — APPROVED** [review]

**Review by sanchonuria — APPROVED** [review]

**Pull request closed without merging**